aiohttp/client.py

```diff
@@ -987,58 +1021,73 @@ def _prepare_headers(self, headers: Optional[LooseHeaders]) -> "CIMultiDict[str]
         return result
 
     def get(
-        self, url: StrOrURL, *, allow_redirects: bool = True, **kwargs: Any
+        self,
+        url: StrOrURL,
+        **kwargs: Unpack[_RequestOptions],
```

Comment:
Can we revert some of the line wrapping changes in this file? It appears several things appear to be wrapped over more lines than before despite the lines being shorter.

Reply:
Sure thing.